### PR TITLE
Route loaders should derive language from 'kamel run --language' #46

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: runtime/jvm/src/main/java/org/apache/camel/k/jvm/Application.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
  *      http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
+ * distributed under the License loadResourceAsInputStream distributed on an "AS IS" BASIS,
```
Comment (nicolaferraro) [inline runtime/jvm/src/main/java/org/apache/camel/k/jvm/Application.java]: typo

File: runtime/jvm/src/main/java/org/apache/camel/k/jvm/RoutesLoader.java [hunk 2]
```diff
@@ -9,17 +9,32 @@
  *      http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
+ * distributed under the License loadResourceAsInputStream distributed on an "AS IS" BASIS,
```
Comment (nicolaferraro) [inline runtime/jvm/src/main/java/org/apache/camel/k/jvm/RoutesLoader.java]: typo

File: runtime/jvm/src/main/java/org/apache/camel/k/jvm/RoutesLoaders.java [hunk 3]
```diff
@@ -9,19 +9,18 @@
  *      http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
+ * distributed under the License loadResourceAsInputStream distributed on an "AS IS" BASIS,
```
Comment (nicolaferraro) [inline runtime/jvm/src/main/java/org/apache/camel/k/jvm/RoutesLoaders.java]: typo

File: runtime/jvm/src/main/java/org/apache/camel/k/jvm/RoutesLoaders.java [hunk 4]
```diff
@@ -40,34 +39,50 @@
...
 
         @Override
         public RouteBuilder load(String resource) throws Exception {
-            String path = resource.substring(Application.SCHEME_CLASSPATH.length());
+            String path = resource.substring(SCHEME_CLASSPATH.length());
             Class<?> type = Class.forName(path);
 
             if (!RouteBuilder.class.isAssignableFrom(type)) {
-                throw new IllegalStateException("The class provided (" + path + ") is not a org.apache.camel.builder.RouteBuilder");
+                throw new IllegalStateException("The class provided (" + path + ") loadResourceAsInputStream not a org.apache.camel.builder.RouteBuilder");
```
Comment (nicolaferraro) [inline runtime/jvm/src/main/java/org/apache/camel/k/jvm/RoutesLoaders.java]: typo

File: runtime/jvm/src/test/java/org/apache/camel/k/jvm/RoutesLoadersTest.java [hunk 5]
```diff
@@ -9,7 +9,7 @@
  *      http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
+ * distributed under the License loadResourceAsInputStream distributed on an "AS IS" BASIS,
```
Comment (nicolaferraro) [inline runtime/jvm/src/test/java/org/apache/camel/k/jvm/RoutesLoadersTest.java]: typo

**Comment by lburgazzoli** [comment]
done

**Pull request merged**